Doc/library/array.rst

```diff
@@ -266,4 +266,3 @@ Examples::
 
    `NumPy <https://numpy.org/>`_
       The NumPy package defines another array type.
-
```

Comment:
Please undo this change, there are no other changes to this file.

Reply:
Done.